am-lib/src/main/java/uk/gov/hmcts/reform/amlib/utils/Permissions.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.amlib.utils;
+
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+import uk.gov.hmcts.reform.amlib.exceptions.UnsupportedPermissionsException;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public final class Permissions {
+
+    private Permissions() {
+
+    }
+
+    public static int sumOf(Set<Permission> perms) {
```

Comment:
This comment might be a bit out of scope but can I ask you to add Javadoc to that public method as well? If not now please create small PR when this story is merged.

Reply:
ofcourse I can :D